packages/block-editor/src/components/block-tools/block-contextual-toolbar.js

```diff
@@ -7,19 +7,74 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { forwardRef, useEffect, useRef, useState } from '@wordpress/element';
 import { hasBlockSupport, store as blocksStore } from '@wordpress/blocks';
 import { useSelect } from '@wordpress/data';
+import {
+	ToolbarItem,
+	ToolbarButton,
+	ToolbarGroup,
+} from '@wordpress/components';
+import { levelUp } from '@wordpress/icons';
+import { useViewportMatch } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
 import NavigableToolbar from '../navigable-toolbar';
 import BlockToolbar from '../block-toolbar';
 import { store as blockEditorStore } from '../../store';
+import BlockIcon from '../block-icon';
+
+const CollapseFixedToolbarButton = forwardRef( ( { onClick }, ref ) => {
+	return (
+		<ToolbarItem
+			as={ ToolbarButton }
```

Comment:
Out of curiosity, have you tried using `ToolbarButton` directly (instead of using the `as` prop) ?

```suggestion
		<ToolbarButton
```

This suggestion would apply for the other instances of `ToolbarItem` in this file too

Reply:
Of course not, I am a robot doing things as instructed 😂 that was `as Button`, suggested use `ToolbarButton` so I did. 🤦🏻